msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -986,7 +987,7 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 |`actions`| Array | | | Specifies the set of actions supported by the runtime. An action is either running a JavaScript function or opening a view such as a task pane.|
 |`actions.id`| String | 64 characters | ✔️ | Specifies the ID for the action, which is passed to the code file. |
 |`actions.type`| String | | ✔️ | Specifies the type of action. The `executeFunction` type runs a JavaScript function without waiting for it to finish and the `openPage` type opens a page in a given view. |
-|`actions.displayName`| String | 64 characters | | Specifies the display name of the action and it isn't the label of a button or a menu item that invokes the action (which is configured with `tabs.groups.controls.label`).|
+|`actions.displayName`| String | 64 characters | | Describes the action of a [custom keyboard shortcut](/office/dev/add-ins/design/keyboard-shortcuts). This description appears in the dialog that's shown to a user when there's a shortcut conflict between multiple add-ins or with Microsoft 365. The `actions.displayName` property is required if a keyboard shortcut is defined for a specific `actions` object. Office appends the name of the add-in in parentheses after the description. For example, "Set bold (Contoso Add-in)". For more information, see [Avoid key combinations in use by other add-ins](/office/dev/add-ins/design/keyboard-shortcuts#avoid-key-combinations-in-use-by-other-add-ins). |
```

Comment:
Hi @samantharamon,
1. Please remove double quotes for "Set bold" as it is not required. 
2. Change capital to small casing for all the instances of links after see. Example, 'For more information, see Avoid" to "For more information, see avoid"